### PR TITLE
Add a hack handling a weird situation where we don't have content.

### PR DESCRIPTION
**Pull request by codeviking**
@aria42

### CODE REVIEW

**Comment by codeviking** [comment]
@aria42 I jammed in another change, it allows the scroll element to be set explicitly.

File: pdf-viewer.ts [hunk 1]
```diff
@@ -49,7 +50,7 @@ module StructuredPDFViewer {
       var sectionY = sectionHeader.yOffset()
       var dy = this.viewportScale * this.panelSkew * (sectionY - panelY)
       var scrollTop = dy - (this.displayParams.sectionJumpVerticalPad || 40)
-      this.displayParams.canvasElem.parentElement.scrollTop = scrollTop
+      this.displayParams.scrollParent.scrollTop = scrollTop
```
Comment (aria42) [inline pdf-viewer.ts:53]: ah yes, good catch

**Comment by aria42** [comment]
lgtm

**Comment by aria42** [comment]
merge at will

**Pull request merged**